lib/filter/filter-call.c

```diff
@@ -49,6 +49,11 @@ filter_call_eval(FilterExprNode *s, LogMessage **msgs, gint num_msg)
   else
     stats_counter_inc(self->super.not_matched);
 
+  msg_debug("  filter() evaluation result",
+            filter_result_tag(res),
+            evt_tag_str("called-rule", self->rule),
+            evt_tag_printf("msg", "%p", msgs[num_msg - 1]));
```

Comment:
This is a little bit confusing for me. Filter call eval works on a list of msgs. Should we build list of evt_tags for all messages here? Is it enough to consider the typical scenario when filter_call_eval is called only for one message?